kernelcare/README.md

```diff
@@ -0,0 +1,54 @@
+# Agent Check: kernelcare
+
+## Overview
+
+This check monitors [kernelcare][1] through the Datadog Agent.
```

Comment:
Can we include a brief description about kernelcare

Reply:
README description improved